### ISSUE TITLE
Problems During Content Assist

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tsaiyilien/google-plugin-for-eclipse#44]
```
What steps will reproduce the problem?
1. Move the cursor to a parameter for a call to a method
2. Hit Ctrl+Shift+Space to get parameter insight
3. An error dialog is displayed
4. Dismissing the error dialog shows the expected parameter insight\
5. Disabling the Google Plugin, or the Basic proposal as suggested by the 
dialog does not resolve the issue.

What is the expected output? What do you see instead?
Parameter insight is the expected output, but the error dialog is displayed 
instead.

What version of the product are you using? On what operating system?
GPE version 2.5.0v201112160242-rel-r37
eclipse.buildId=M20110909-1335 (Indigo SR1), with everything up to date
Windows 7 64bit edition
java.version=1.6.0_29

Please provide any additional information below.
I doubt it's relevant, but I recently upgraded from Java 1.6.0_26 to 1.6.0_29. 
I don't recall seeing the issue prior to 1.6.0_29
Other plugins installed include Window Builder, Subclipse, Checkstyle, and 
Mylyn including the JIRA connector
```

Original issue reported on code.google.com by `wingnu...@gmail.com` on 10 Jan 2012 at 3:27

Attachments:
- [ParameterInsightError.png](https://storage.googleapis.com/google-code-attachments/google-plugin-for-eclipse/issue-44/comment-0/ParameterInsightError.png)

**Comment by GoogleCodeExporter** [comment]
```
Looks like someone else already reported it.. can't believe I didn't see the 
issue when first scanning over the list.
http://code.google.com/p/google-plugin-for-eclipse/issues/detail?id=42
```

Original comment by `wingnu...@gmail.com` on 10 Jan 2012 at 7:42

**Comment by GoogleCodeExporter** [comment]
```
If this is of any assistance... I just created a new workspace from scratch, 
still shows up.. so doesn't appear to be as a result of some funny workspace 
state etc.
```

Original comment by `wingnu...@gmail.com` on 11 Jan 2012 at 2:36

**Comment by GoogleCodeExporter** [comment]
```
I have this issue when I install Eclipse Indigo for Java, Android Development 
Tools and Google App Engine (no other plugins).
This shows up both in Mac OS X 10.7 x64 and Windows 7 x64
```

Original comment by `timelza...@gmail.com` on 18 Jan 2012 at 11:16

**Comment by GoogleCodeExporter** [comment]
```
I have the same issue.
OS: Ubuntu 11.04 (x32)
Eclipse: Indigo Service Release 1 Build id: 20110916-0149

```

Original comment by `ale.sch...@gmail.com` on 14 Feb 2012 at 4:18

**Comment by GoogleCodeExporter** [comment]
```
We are working on this
```

Original comment by `chintanr...@gmail.com` on 28 Feb 2012 at 4:32

**Comment by GoogleCodeExporter** [comment]
```
I just did a "Check for Updates" within Eclipse, and a new version of the 
plugin got installed. The problem seems to have disappeared for me.

Has this bug been fixed, but yet to be marked as closed?, or has it been fixed 
as a positive side-effect of another bug fix?

Anyhow, it's great now that code-completion/parameter insight works properly 
for me again. Cheers
```

Original comment by `wingnu...@gmail.com` on 1 Mar 2012 at 9:00

**Comment by GoogleCodeExporter** [comment]
```
fixed in 2.5.2. Please check.
```

Original comment by `chintanr...@google.com` on 7 Mar 2012 at 12:32
- Changed state: **Fixed**
